src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -1186,6 +1187,8 @@ public function unitAction($action, $id)
      * @param int $productId
      *
      * @return JsonResponse
+     *
+     * @todo: move/copy this action to product V2 controller
```

Comment:
Please create an issue and link this @todo to the issue.

Reply:
https://github.com/PrestaShop/PrestaShop/issues/28397

Reply:
yeah I don't remember what was about this reminder, probably not relevant anymore